test/compile/common.test.ts

```diff
@@ -3,94 +3,104 @@
 import {assert} from 'chai';
 import {parseUnitModel} from '../util';
 import {X} from '../../src/channel';
-import {timeFormat, formatMixins, applyColorAndOpacity} from '../../src/compile/common';
+import {formatMixins, applyColorAndOpacity} from '../../src/compile/common';
+import {format} from '../../src/timeunit';
 
 describe('Model', function() {
-  describe('timeFormat()', function() {
+  describe('format()', function() {
```

Comment:
Yep. If there isn't one yet.  

Make sure you add the new file's path to `tsconfig.json`  :)